concepts/Common/Integration Guides/60 Manage Amazon S3 Storage with DevExtreme Components/10 Amazon Setup.md

```diff
@@ -0,0 +1,38 @@
+### Create a New Bucket
+
+**Main article**: [Amazon Documentation — Creating a Bucket](https://docs.aws.amazon.com/AmazonS3/latest/userguide/create-bucket-overview.html)
+
+**Additional video tutorials**: [@TinyTechnicalTutorials](https://youtu.be/mDRoyPFJvlU?t=360), [@CodeWithOtabek](https://www.youtube.com/watch?v=BSZqR0SXf3g)
```

Comment:
Similar to my comment above, shouldn't this link to the video tutorials go at the end of the section, after your explanation?